src/models/fhir/index.ts

```diff
@@ -2,3 +2,5 @@ import { parse } from "./parse";
 import { hasRequiredFields } from "./required";
 
 export default { parse, hasRequiredFields };
+
+export * from "./maskNricFin";
```

Comment:
Would it make more sense to move `src/models/fhir/maskNricFin.ts` -> `src/common/maskNricFin.ts`? Cause `maskNricFin.ts` isn't exactly making changes to the FHIR helper functions.